data/vaults/1/0x8b9C0c24307344B6D7941ab654b2Aeee25347473.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "vault",
+  "comment": "Curve EURN Pool",
+  "hideAlways": false,
+  "depositsDisabled": false,
+  "withdrawalsDisabled": false,
+  "order": 37,
+  "apyTypeOverride": false,
+  "migrationAvailable": false,
+  "allowZapIn": false,
+  "allowZapOut": false,
```

Comment:
Can you set them to `true` so we can test in prod if zaps fail. If zaps fail, we can turn them off again here.